pylint/test/functional/deprecated_methods_py36.py

```diff
@@ -0,0 +1,40 @@
+""" Functional tests for method deprecation. """
```

Comment:
I did a full copy of the tests to account for inspect.getmoduleinfo being removed from 3.6. 
The two options I can see are splitting the inspect.getmoduleinfo line into it's own version dependent test (changing the old test), or moving to a per version list of everything that is depreciated (duplicating tests).

If you confirm that you like the first version better, I'll update the pull request.